easybuild/easyconfigs/d/DIAMOND/DIAMOND-0.9.30-GCC-8.3.0.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'CMakeMake'
```

Comment:
@lexming Please reinstate the original attribution, then add a comment to note your own changes.

Reply:
I updated `DIAMOND-0.9.24-iccifort-2019.1.144-GCC-8.2.0-2.31.1.eb` which does not have any attribution to it.

Reply:
@lexming sure, but if you look at the `foss` versions it was still there. Not a vital thing, but I think it's a shame to remove these attributions.

Reply:
I see, I don't mind adding back the original attribution but I won't add any other comment attributing changes to authors as I cannot track all the updates to the various easyconfigs of `DIAMOND` in the past years.